module.py

```diff
@@ -562,16 +564,25 @@ def handle_op(self, codeobj, context, label, op, oparg, line):
             context.insert_handle_error(line, label)
             context.end_block()
 
-            for i in range(oparg):
-                context.insert_line('v = PyTuple_GET_ITEM(x, %d);' % (oparg - i - 1))
-                context.insert_line('w = POP();')
-                context.insert_line('err = PyDict_SetItem(u, v, w);')
-                context.insert_line('Py_DECREF(w);')
-                context.insert_line('if (err != 0)')
-                context.begin_block()
-                context.insert_line('Py_DECREF(u);')
-                context.insert_handle_error(line, label)
-                context.end_block()
+            context.insert_line('for (i = %d; i > 0; i--)' % oparg)
```

Comment:
Nowhere in generated code we have loops. Because `oparg` is known at compile time, it's preferable to unroll the loop. In practice, keep the `for i in range(oparg)` loop but do it backwards to get the items inserted in the order we need.

Reply:
> Nowhere in generated code we have loops.

But then there's this [loop that's generated for the `BUILD_MAP` opcode which also uses `oparg`](https://github.com/Pypperoni/pypperoni/blob/732050a046edde0d97a5b2fe7042285baccce67d/module.py#L519).  I've actually written the new code without the loop prior to committing this, but then I saw how `BUILD_MAP` gets processed, and I thought it's safe to do so here for `BUILD_CONST_KEY_MAP`, but yeah I'll revert to the python loop.